tests/ConfigTest.php

```diff
@@ -42,56 +46,60 @@ public function testConfigurationDisableAutowire()
         $config->configureContainer($builder);
     }
 
-    public function testConfigurationKeysValues()
+    public function testConfigurationKeysValues(): void
     {
-        $config = ['a' => new \DateTime(), 'b' => [1, 2, 3], 'c' => 'd'];
+        $config = ['a' => new DateTime(), 'b' => [1, 2, 3], 'c' => 'd'];
 
         $container = $this->getContainer($config);
 
         $config = $container->get(Config::CONFIG);
 
         self::assertNotEmpty($config);
-        self::assertInstanceOf(\DateTime::class, $config['a']);
+        self::assertInstanceOf(DateTime::class, $config['a']);
         self::assertSame([1, 2, 3], $config['b']);
         self::assertSame('d', $config['c']);
     }
 
-    public function testConfigurationEnableCompilation()
+    public function testConfigurationEnableCompilation(): void
     {
-        vfsStream::setup('project');
-
-        $url = vfsStream::url('project');
```

Comment:
$url    = sys_get_temp_dir(); better approach and you don't need to modify gitignore and delete the file: 
check my correction: https://travis-ci.com/github/elie29/zend-di-config/builds/220737669